Datawheel/canon
# [test] status API

## New Package
I created a new package  called `canon-test` following the structure of the current packages and pnpm rules.
This new package is a super simple express app.

### To test it, run:
- `pnpm i` in root.
- Create a basic `.env` in `packages/test` -the same as you use in the examples page-. Remember to `source .env`.
- Terminal 1: `pnpm run dev` in `packages/test`.
- Terminal 2: `pnpm run dev` in `examples`.
- Go to: http://localhost:3000/status and see the results of the status test.

## Small changes in other packages
### CMS
- I created a new custom api `/api/alive/cms` that for now just return a `true` status. Could be more comprehensive in the future.

### Core
- I created a new custom api `/api/alive/core` that for now just return a `true` status. Could be more comprehensive in the future.

## Publish the package
Could you help me with publishing the new package?, I'm not sure how to use the `release.js`.
I need the new version numbers to test it in a real project. At least as a pre-release. I couldn't find a way to reference a dependency from a branch of a package that doesn't exists.

Thanks!
